application/__init__.py

```diff
@@ -46,6 +47,17 @@
 else:
     app = Flask(__name__, static_folder="../client/build")
 
+
+@app.errorhandler(Exception)
```

Comment:
Excellent! This avoids us explicitly logging every error in route handlers.

Can we also use add a `@app.after_request` handler to automatically log the responses? (We might need some special handling to avoid logging sensitive information like a route that parse VNC passwords, but we can think more about those in anther PR. )